gateway/src/main.rs

```diff
@@ -19,16 +20,15 @@ static GLOBAL: MiMalloc = MiMalloc;
 async fn main() {
     // Set up logs and metrics
     observability::setup_logs();
-    let metrics_handle = observability::setup_metrics().expect_pretty("Failed to set up metrics");
+    let metrics_handle = observability::setup_metrics().ok_or_log();
```

Comment:
This should still fail i think, nothing to do with ClickHouse. (And generally shouldn't even fail.)